### ISSUE TITLE
Allow user to add multiple instances of same visualisation to dashboard

### ISSUE THREAD

**Issue by FredrikAugust** [IT2901-SINTEF01/frontend#139]
### Description
Add several instances of same visualisation of same data set.

### Subtasks
- [x] Create unique ID for each dashboard visualisation for use in React (UUID should suffice)
- [x] Use ID as the unique part, and not the DATASOURCE-VISTYPE template type

**Comment by Kpaubert** [comment]
Perhaps we could change the data format to be something like this:
```json
{
  "dashboard": [
    {
      "6dd4e44e-86b7-4fac-9074-0773a48b9945": {
        "MET_API-thresholdchart": {
          "visualisationType": "thresholdchart",
          "dataSourceId": "MET_API",
          "options": {
            "size": 6
          }
        }
      }
    },
    {
      "b7fbd543-4191-4f30-ba49-7bb980aae5a4": {
        "MET_API-linechart": {
          "visualisationType": "linechart",
          "dataSourceId": "MET_API",
          "options": {
            "size": 6
          }
        }
      }
    }
  ]
}
```

Do you think that would suffice? Not sure how much would need to be changed for this to work, but I can look into it if it looks feasible. @FredrikAugust

**Comment by FredrikAugust** [comment]
The top-level nesting seems unnecessary. ID>{ type: ..., ... } should suffice?

**Comment by FredrikAugust** [comment]
Closed by #145
